collect_app/src/main/java/org/odk/collect/android/preferences/PreferenceKeys.java

```diff
@@ -3,107 +3,121 @@
+        hashMap.put(KEY_USERNAME,                   "");
+        // form_management_preferences.xml
+        hashMap.put(KEY_AUTOSEND,                   AUTOSEND_OFF);
         hashMap.put(KEY_DELETE_AFTER_SEND,          false);
-        hashMap.put(KEY_ANALYTICS,                  true);
+        hashMap.put(KEY_COMPLETED_DEFAULT,          true);
+        hashMap.put(KEY_CONSTRAINT_BEHAVIOR,        CONSTRAINT_BEHAVIOR_ON_SWIPE);
+        hashMap.put(KEY_HIGH_RESOLUTION,            true);
+        hashMap.put(KEY_IMAGE_SIZE,                 "original_image_size");
         hashMap.put(KEY_INSTANCE_SYNC,              true);
-        hashMap.put(KEY_APP_LANGUAGE,               "");
-        hashMap.put(KEY_PROTOCOL,                   Collect.getInstance().getString(R.string.protocol_odk_default));
-        hashMap.put(KEY_SELECTED_GOOGLE_ACCOUNT,    "");
-        hashMap.put(KEY_USERNAME,                   "");
+        // form_metadata_preferences.xml
         hashMap.put(KEY_METADATA_USERNAME,          "");
         hashMap.put(KEY_METADATA_PHONENUMBER,       "");
         hashMap.put(KEY_METADATA_EMAIL,             "");
-        hashMap.put(KEY_SERVER_URL,                 Collect.getInstance().getString(R.string.default_server_url));
+        // google_preferences.xml
+        hashMap.put(KEY_SELECTED_GOOGLE_ACCOUNT,    "");
         hashMap.put(KEY_GOOGLE_SHEETS_URL,          "");
+        // identity_preferences.xml
+        hashMap.put(KEY_ANALYTICS,                  true);
+        // other_preferences.xml
         hashMap.put(KEY_FORMLIST_URL,               Collect.getInstance().getString(R.string.default_odk_formlist));
         hashMap.put(KEY_SUBMISSION_URL,             Collect.getInstance().getString(R.string.default_odk_submission));
-        hashMap.put(KEY_IMAGE_SIZE,                 "original_image_size");
+        // server_preferences.xml
+        hashMap.put(KEY_PROTOCOL,                   Collect.getInstance().getString(R.string.protocol_odk_default));
+        // user_interface_preferences.xml
+        hashMap.put(KEY_APP_LANGUAGE,               "");
+        hashMap.put(KEY_FONT_SIZE,                  Collect.DEFAULT_FONTSIZE);
         hashMap.put(KEY_NAVIGATION,                 NAVIGATION_SWIPE);
-        hashMap.put(KEY_CONSTRAINT_BEHAVIOR,        CONSTRAINT_BEHAVIOR_ON_SWIPE);
-        hashMap.put(KEY_COMPLETED_DEFAULT,          true);
+        hashMap.put(KEY_SHOW_SPLASH,                false);
+        hashMap.put(KEY_SPLASH_PATH,                Collect.getInstance().getString(R.string.default_splash_path));
         hashMap.put(KEY_MAP_SDK,                    GOOGLE_MAPS);
         hashMap.put(KEY_MAP_BASEMAP,                GOOGLE_MAPS_BASEMAP_DEFAULT);
-        hashMap.put(KEY_AUTOSEND,                   AUTOSEND_OFF);
         return hashMap;
     }
 
+    static Collection<String> keysWeShouldNotReset = Arrays.asList(
```

Comment:
Can this be `final`? Can we also give it a `CONSTANT` name if so? If not, can we document under what condition one should modify the field?